src/main/java/net/snowflake/ingest/utils/ParameterProvider.java

```diff
@@ -253,6 +263,16 @@ private void setParameterMap(
         props,
         false);
 
+    this.checkAndUpdate(
+        PARQUET_WRITER_VERSION, PARQUET_WRITER_VERSION_DEFAULT, parameterOverrides, props, false);
+
+    this.checkAndUpdate(
+        ENABLE_PARQUET_DICTIONARY_ENCODING,
+        ENABLE_PARQUET_DICTIONARY_ENCODING_DEFAULT,
+        parameterOverrides,
+        props,
+        false);
```

Comment:
nit: add inline comment for argument name

Reply:
Added.